registry_resolver/build.rs

```diff
@@ -1,14 +1,63 @@
-fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let package_path = "buf.build/knox-networks/registry-mgmt";
-    let commit = "43801fd1632e40668c5ae75110550d6d";
-    let std::process::Output { status, stderr, .. } = std::process::Command::new("buf")
-        .arg("generate")
-        .arg(format!("{package_path}:{commit}"))
-        .arg("--output")
-        .arg(std::env::var("OUT_DIR").unwrap())
-        .output()?;
+use bigerror::{BuildError, Report, ReportAs, Reportable};
+use std::{env, path::PathBuf};
+
+fn main() -> Result<(), Report<BuildError>> {
+    let out_dir = env::var("OUT_DIR").unwrap();
+
+    let proto_dir = PathBuf::from(&out_dir).join("protos");
+
+    let std::process::Output {
+        status,
+        stderr,
+        stdout,
+        ..
+    } = std::process::Command::new("protofetch")
+        .env("RUST_LOG", "TRACE")
+        .arg("--output-proto-directory")
+        .arg(&proto_dir)
+        .arg("fetch")
+        .output()
+        .report_as()?;
+
+    println!(
+        "protofetch stdout: {}",
+        String::from_utf8_lossy(&stdout).into_owned()
+    );
+    println!(
+        "protofetch stderr: {}",
+        String::from_utf8_lossy(&stderr).into_owned()
+    );
+
     if !status.success() {
-        return Err(String::from_utf8_lossy(&stderr).into());
+        return Err(BuildError::with_kv(
+            "error_msg",
+            String::from_utf8_lossy(&stderr).into_owned(),
+        ));
```

Comment:
```suggestion
        return Err(BuildError::attach(String::from_utf8_lossy(&stderr).into_owned()));
```
Not sure we need a kv here